learning-outcomes.md

```diff
@@ -0,0 +1,50 @@
+# Week 4 Learning Outcomes
+
+## Node
+
+### Servers
+
+- [ ] **We can create a web server that responds to http requests**
+- [ ] We can conditionally set ports for our server based on the runtime environment
+- [ ] We can read secrets from a .env file
+- [ ] We can prevent sensitive data from being pushed to GitHub
+
+### GET requests
+
+- [ ] We can route requests to the correct handler function(s)
+- [ ] We can serve different types of files to the client
+- [ ] We can read information sent in a URL query string
+
+### POST requests
+
+- [ ] We can handle data received from a POST request
+- [ ] We can ‘chunk’ data sent from the client side on the server side
```

Comment:
This might be misleading—the request body arrives in chunks that we have to handle. We don't actually chunk the data ourselves

```suggestion
- [ ] We can handle streams of data sent from the client that don't arrive all at once
```